_dashboards/dev-tools/run-queries.md

```diff
@@ -1,33 +1,23 @@
 ---
 layout: default
-title: Running queries in the Dev Tools console
+title: Running queries in the Dev Tools Console
```

Comment:
It looks like we actually don't capitalize "console" above.

Reply:
Changed to lowercase across docs